### PR TITLE
Convert knockout typeahead binding from jQuery UI to atwho

### PR DESCRIPTION
**Pull request by orangejenny**
Same rationale as https://github.com/dimagi/langcodes/pull/8

@calellowitz  / @emord

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/apps/style/static/style/js/atwho.js, line 3 - 'otherEvent' is defined but never used (no-unused-vars)
* corehq/apps/style/static/style/js/atwho.js, line 26 - 'searchKey' is defined but never used (no-unused-vars)
* corehq/apps/style/static/style/js/atwho.js, line 31 - 'should_startWithSpace' is defined but never used (no-unused-vars)
* corehq/apps/style/static/style/js/atwho.js, line 34 - '$li' is defined but never used (no-unused-vars)
* corehq/apps/style/static/style/ko/knockout_bindings.ko.js, line 497 - Expected indentation of 12 space characters but found 11. (indent)

File: corehq/apps/style/static/style/js/atwho.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+     */
+    var init = function($input, ajaxOptions, callback) {
+        var atwhoOptions = {
+            at: "",
+            limit: Infinity,
+            maxLen: Infinity,
+            suffix: "",
+            tabSelectsMatch: false,
+            callbacks: {
+                filter: function(query, data, searchKey) {
```
Comment (dimagimon) [inline corehq/apps/style/static/style/js/atwho.js]: 'searchKey' is defined but never used (no-unused-vars)

**Comment by orangejenny** [comment]
Ignoring dimagimon.

**Review by emord — APPROVED** [review]
Seems reasonable to me although I'm not super familiar with knockout

**Pull request merged**